src/bentoml/_internal/server/runner_app.py

```diff
@@ -68,11 +69,14 @@ def fallback():
 
         for method in runner.runner_methods:
             max_batch_size = method.max_batch_size if method.config.batchable else -1
-            self.dispatchers[method.name] = CorkDispatcher(
-                max_latency_in_ms=method.max_latency_ms,
-                max_batch_size=max_batch_size,
-                fallback=fallback,
-            )
+            if method.config.is_stream:
+                self.dispatchers[method.name] = NoOpDispatcher()
+            else:
+                self.dispatchers[method.name] = CorkDispatcher(
+                    max_latency_in_ms=method.max_latency_ms,
+                    max_batch_size=max_batch_size,
+                    fallback=fallback,
+                )
```

Comment:
let's also make this a bit more clean. Can we make it similar to how we dispatch the class for runner_handle?